routes/blog/index.tsx

```diff
@@ -36,6 +37,12 @@ export default function BlogIndexPage(props: PageProps<Post[]>) {
     <>
       <Head>
         <title>Blog</title>
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content={SITE_NAME} />
+        <meta property="og:locale" content="en" />
+        <meta property="og:title" content="Blog" />
+        <meta property="og:url" content={props.url.origin} />
+        <meta name="twitter:title" content="Blog" />
```

Comment:
Can we match the number of tags as in `/blog/[slug].tsx`? We can have something like "This is the blog for Deno SaaSKit" for the description.

Reply:
Sure, added it in the new PR.